### PR TITLE
Prod Deployment Update

### PR DESCRIPTION
**Pull request by Encodian**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by Encodian** [comment]
Requested update complete

File: certified-connectors/Encodian/Readme.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+
+## Encodian for PowerAutomate
```
Comment (dougelliott) [inline certified-connectors/Encodian/Readme.md:2]: Space needed in "PowerAutomate" to "Power Automate"

File: certified-connectors/Encodian/Readme.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+[Encodian for PowerAutomate](https://www.encodian.com/products/encodian-for-microsoft-power-automate/) - Supercharge your Microsoft Power Automate and Azure Logic Apps solutions with our enhanced document management actions, providing PDF conversion, document merging, watermarks, OCR and advanced document and image clean-up capabilities..
```
Comment (dougelliott) [inline certified-connectors/Encodian/Readme.md:3]: Space needed here as well. Thanks!

Comment (Encodian) [reply certified-connectors/Encodian/Readme.md]: All done, pushed to the master branch on  Encodian/PowerPlatformConnectors

Comment (dougelliott) [reply certified-connectors/Encodian/Readme.md]: Thanks!

**Pull request merged**